### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by TheiLLeniumStudios**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Does your code fit the style guidelines? [yes/no]
- Does your PR fit the contribution guidelines? [yes/no]

### CODE REVIEW

**Comment by IdrisDose** [comment]
@TheiLLeniumStudios bump plz fix checks ;)

**Comment by TheiLLeniumStudios** [comment]
@IdrisDose Checks are passing now

File: client/main.lua [hunk 1]
```diff
@@ -293,8 +294,8 @@ CreateThread(function()
             local hours = GetClockHours()
             if hours >= Config.MinimumTime or hours <= Config.MaximumTime then
                 if not inside then
-                    for k, v in pairs(Config.Houses) do
-                        dist = #(PlayerPos - vector3(Config.Houses[k]["coords"]["x"], Config.Houses[k]["coords"]["y"], Config.Houses[k]["coords"]["z"]))
+                    for k, _ in pairs(Config.Houses) do
```
Comment (BerkieBb) [inline client/main.lua:297]: You can remove this underscore completely

File: client/main.lua [hunk 2]
```diff
@@ -341,7 +342,7 @@ CreateThread(function()
                 end
             end
 
-            for k, v in pairs(Config.Houses[currentHouse]["furniture"]) do
+            for k, _ in pairs(Config.Houses[currentHouse]["furniture"]) do
```
Comment (BerkieBb) [inline client/main.lua:345]: You can remove this underscore completely

File: client/main.lua (hunk 1 above)
Comment (IdrisDose) [reply client/main.lua:297]: The underscore is there to indicate that it is not used, why would it be removed?

**Review by IdrisDose — APPROVED** [review]

**Pull request merged**